index.html

```diff
@@ -1,47 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Document</title>
   <link rel="stylesheet" href="./Styles/styles.css" />
-  <link rel="stylesheet"
-    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
 </head>
 
 <body>
+
+  <!-- header section -->
+
   <header>
     <a href="index.html" class="logo">
       <h1>Byiringiro Moise</h1>
     </a>
     <span class="material-symbols-outlined"> menu </span>
+    <div class="nav-lists">
+      <a href="">Profile</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+      <a href=""><span class="material-symbols-outlined"> mail </span></a>
+    </div>
```

Comment:
- [ ] [REQUIRED] Your navbar navigation items are missing. I noticed it was present when you submitted it for the first review. But now, it is gone. Kindly look into this and fix it. 
![image](https://user-images.githubusercontent.com/84156714/193332551-14f7f880-00c3-4164-827d-26a0c67bdff4.png)